lib/pf/util.pm

```diff
@@ -1250,6 +1267,19 @@ sub validate_date {
     return $TRUE;
 }
 
+=item clean_locale
+
+Clean the format of the locale stored
+
+=cut
+
+sub clean_locale {
+    my ($locale) = @_;
+    if( $locale =~ /^([A-Za-z_]+)\./ ) {
+        $locale = $1;
+    }
```

Comment:
i prefer to have something more explicit like:
sub clean_locale {
    my ($locale) = @_;
    if( $locale =~ /^([A-Za-z_]+)\./ ) {
        return $1;
     }
    return $locale;
}